src/tests/suite/daos_test.h

```diff
@@ -90,7 +90,15 @@ struct test_pool {
 	daos_handle_t		poh;
 	daos_pool_info_t	pool_info;
 	daos_size_t		pool_size;
+	/* Updated if some ranks are killed during degraged or rebuild
+	 * test, so we know whether some tests is allowed to be run.
+	 */
 	d_rank_list_t		svc;
+	/* Used for all pool related operation, since client will
+	 * use this rank list to find out the real leader, so it
+	 * can not be changed.
+	 */
+	d_rank_list_t		orig_svc;
```

Comment:
keep both svc and orig_svc seems quite confuse -- when to use which svc...
maybe need not add orig_svc, but add another variable of "int svc_alive_nr" to test_pool, and 
1) in daos_exclude_server() to reduce it, 
2) in the test code check arg->poo.svc_alive_nr rather than arg->pool.svc.rl_nr to check if the alive raft nodes is enough, or add another helper like "dts_alive_raft_node_nr(test_arg_t *arg)".

Reply:
ok, sounds good